modules/Galleries/Galleries.module.scss

```diff
@@ -1,3 +1,5 @@
-.loadMore {
+// TODO: Figure out how to decrease the specificity of styles from the UI library
+.loadMore.loadMore {
```

Comment:
Did you consider using [`mini-css-extract-plugin`](https://github.com/webpack-contrib/mini-css-extract-plugin) in the theme component package?

That would build one stylesheet for the entire package which you would then import in the beginning of Bea theme and then you should not have any problems with this? 🤔 I am not 100 % sure though.

Reply:
I did, but as I mentioned in the PR description it will bloat the resulting bundle (no tree-shaking / dynamic imports).
I believe I can also tell the style-inject rollup plugin to inject the styles at the top of the `head`, this might solve the order issue.

Reply:
Note that I'm using Rollup here, not Webpack :)